lib/components/icons/Icon/Icon.tsx

```diff
@@ -42,7 +42,7 @@ export const Icon = ({
       className={classnames(
         resolveSizeClasses(size, inline),
         styles.currentColor,
-        fill ? useTextColor(fill) : null,
+        useTextColor(fill),
```

Comment:
I didn't even realise that this fixes a bug where icons didn't have a `fill` set by default, so they ended up rendering with a black fill rather than `neutral`.

You can thank Chromatic for pointing this out 😄